client/extensions/woocommerce/woocommerce-services/state/shipping-label/reducer.js

```diff
@@ -144,6 +153,13 @@ reducers[ WOOCOMMERCE_SERVICES_SHIPPING_LABEL_EXIT_PRINTING_FLOW ] = ( state, {
 	};
 };
 
+reducers[ WOOCOMMERCE_SERVICES_SHIPPING_EXIT_TRACKING_FLOW ] = ( state ) => {
```

Comment:
I don't have a strong opinion here, but we should probably remove the parenthesis here around the `state` argument, like the rest of reducers in this file with only one argument.